recipes/category_encoders/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set version = "1.2.1" %}
+{% set gh_org = "wdm0006" %}
+{% set gh_repo = "categorical_encoding" %}
+{% set pkg_name = "category_encoders" %}
+
+
+package:
+  name: {{ pkg_name }}
+  version: {{ version }}
+
+source:
+  fn: {{ gh_repo }}-{{ version }}.tar.gz
+  url: https://github.com/{{ gh_org }}/{{ gh_repo }}/archive/{{ version }}.tar.gz
+  sha256: d4649bf425c4398c9ed54e5d4021181d652e05189242d823658e6fad9b986c2f
+
+build:
+  number: 0
+  script:
+    - python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - numpy >=1.8.0
+    - pandas >=0.15.0
+    - patsy >=0.4.0
+    - python
+    - scikit-learn >=0.15.0
+    - scipy >=0.9
+    - statsmodels >=0.6.0
+
+test:
+  # FIXME: in a future version, tests will run on windows!
+  # https://github.com/wdm0006/categorical_encoding/pull/8
+  requires:
+    - nose # unix
```

Comment:
Yeah, though it could be a legit comment in some cases. 😞 Still no harm in asking. See this issue ( https://github.com/conda-forge/conda-smithy/issues/244 ).